addon/src/test-support/helpers/drag.js

```diff
@@ -108,4 +108,11 @@ export async function drag(mode, itemSelector, offsetFn, callbacks = {}) {
     await callbacks.dragend();
     await settled();
   }
+
```

Comment:
you could also try 

```
await new Promise(requestAnimationFrame)
await settled();
```
if there is an un-instrumented request animation frame in this library.


but it seems like we want to start a waiter when we start dragging and then "endAsync" when we stop dragging.

like:

```js
import { buildWaiter } from '@ember/test-waiters';


const waiter = buildWaiter('ember-sortable:dragging');

// ...

// dragstart:
this.waiter = waiter.beginAsync();

// dragend:
waiter.endAsync(this.waiter);

```

or something like that -- then arbitrary waiting would never need to happen _and_ if something goes wrong with the dragging, we have tools to find out exactly where:
- getPendingWaiterState()
- getSettledState()

(I usually assign both of those to `window` in my test-helper.js, so I can invoke them from the console if I run in to a hanging test)